### PR TITLE
Add feature-rearchitecture branch for odh operator

### PR DESCRIPTION
**Pull request by VaishnaviHire**
- Add `feature-rearchitecture` branch for opendatahub-operator repo.
- Add `feature-rearchitecture` odh-manifests bundle to the PR image

### CODE REVIEW

File: core-services/prow/02_config/opendatahub-io/opendatahub-operator/_prowconfig.yaml [hunk 1]
```diff
@@ -2,7 +2,10 @@ tide:
   merge_method:
     opendatahub-io/opendatahub-operator: squash
   queries:
-  - labels:
+  - includedBranches:
+    - feature/rearchitecture
```
Comment (zdtsw) [inline core-services/prow/02_config/opendatahub-io/opendatahub-operator/_prowconfig.yaml:6]: wonder if here uses "feature/rearchitecture" then then file name need to match it. 
so if we have a branch "feature-rearchitecture" which should be fine.

**Comment by VaishnaviHire** [comment]
@zdtsw Can you add lgtm label ?

**Review by anishasthana — COMMENTED** [review]
/lgtm

**Review by anishasthana — APPROVED** [review]

**Review by zdtsw — APPROVED** [review]
/lgtm

**Comment by VaishnaviHire** [comment]
Hi @jmguzik, do these changes in prow config look good to you?

**Comment by harshad16** [comment]
/lgtm

:+1:

**Pull request merged**